pkg/scheduler/algorithm/scheduler_interface.go

```diff
@@ -41,6 +44,26 @@ type SchedulerExtender interface {
 
 	// IsBinder returns whether this extender is configured for the Bind method.
 	IsBinder() bool
+
+	// ProcessPreemption returns nodes with their victim pods processed by extender based on
+	// given:
+	//   1. Pod to schedule
+	//   2. Candidate nodes and victim pods (nodeToVictims) generated by previous scheduling process.
+	//   3. NodeInfo map for preemption logic to use.
+	//   4. PodDisruptionBudget list.
+	// The possible changes made by extender may include:
+	//   1. Subset of given candidate nodes after preemption phase of extender.
+	//   2. A different set of victim pod for every given candidate node after preemption phase of extender.
+	// If you do not want to implement preemption, just return nodeToVictims.
+	ProcessPreemption(
```

Comment:
Nit: Should we call this Preempt to be consistent with other calls?

Reply:
We intentionally chose a different name to make it easier to track in scheduler code. Let's keep it.

Reply:
Ok